### PR TITLE
[SDL-0273] Processing PTU with WEB_VIEW HMI type

### PR DESCRIPTION
**Pull request by VjKlepikov**
Jira task: https://adc.luxoft.com/jira/browse/FORDTCN-6818

ATF Test Scripts to check processing PTU with WEB_VIEW HMI type

This PR is **[ready]** for review.

### Summary
- Check that PTS is created with appHmiType WEB_VIEW
- Processing PTU with AppHMIType WEB_VIEW

### CODE REVIEW

File: test_scripts/WebEngine/WebEngineProjection/011_Verify_PTS_with_appHMIType_WEB_VIEW.lua [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+--   a. PTU is triggered, SDL sends UPDATE_NEDDED to HMI
+--   b. PTS is created with AppHMIType WEB_VIEW for App1 and other mandatory pts
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared Libraries ]]
+local common = require('test_scripts/WebEngine/commonWebEngine')
+
+--[[ Local Variables ]]
+local appSessionId1 = 1
+local appSessionId2 = 2
+local appHMIType = "WEB_VIEW"
```
Comment (YarikMamykin) [inline test_scripts/WebEngine/WebEngineProjection/011_Verify_PTS_with_appHMIType_WEB_VIEW.lua:23]:
```suggestion
local appHMIType = { "WEB_VIEW" }
```

File: test_scripts/WebEngine/WebEngineProjection/011_Verify_PTS_with_appHMIType_WEB_VIEW.lua [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+--[[ General configuration parameters ]]
+config.application1.registerAppInterfaceParams.appHMIType = { appHMIType }
```
Comment (YarikMamykin) [inline test_scripts/WebEngine/WebEngineProjection/011_Verify_PTS_with_appHMIType_WEB_VIEW.lua:26]: Please remove bracers
```suggestion
config.application1.registerAppInterfaceParams.appHMIType = appHMIType
```
Related to other places in this and other scripts

Comment (VjKlepikov) [reply test_scripts/WebEngine/WebEngineProjection/011_Verify_PTS_with_appHMIType_WEB_VIEW.lua]: @YarikMamykin  Done in  ([77cc64b](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/36/commits/77cc64bc938f0db605c2ba6024ef429f9d6ca497))

File: test_scripts/WebEngine/WebEngineProjection/011_Verify_PTS_with_appHMIType_WEB_VIEW.lua [hunk 3]
```diff
@@ -0,0 +1,57 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0273-webengine-projection-mode.md
+--
+-- Description:
+-- Check that PTS is created with appHmiType WEB_VIEW
+--
+-- Precondition:
+-- 1. SDL and HMI are started
+-- 2. WebEngine App1 with WEB_VIEW HMI type is registered
```
Comment (YarikMamykin) [inline test_scripts/WebEngine/WebEngineProjection/011_Verify_PTS_with_appHMIType_WEB_VIEW.lua:9]: Add precondition step about update of preloaded json. That application has record in preloaded table and has appropriate HMI type.

Comment (VjKlepikov) [reply test_scripts/WebEngine/WebEngineProjection/011_Verify_PTS_with_appHMIType_WEB_VIEW.lua]: Done in [a6a6ab7](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/36/commits/a6a6ab72c20c10ba5a85c2ee566a1b2db7c55b87)

File: test_scripts/WebEngine/WebEngineProjection/011_Verify_PTS_with_appHMIType_WEB_VIEW.lua [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+--
+-- Precondition:
+-- 1. SDL and HMI are started
+-- 2. WEB_VIEW appHMITypes is allowed by policy for application (App1)
+-- 3. WebEngine App1 with WEB_VIEW HMI type is registered
+
+-- Sequence:
+-- 1. App2 register with NAVIGATION HMI type
+-- SDL does:
+--   a. PTU is triggered, SDL sends UPDATE_NEDDED to HMI
```
Comment (LitvinenkoIra) [inline test_scripts/WebEngine/WebEngineProjection/011_Verify_PTS_with_appHMIType_WEB_VIEW.lua:15]:
```suggestion
--   a. PTU is triggered, SDL sends UPDATE_NEEDED to HMI
```

**Pull request merged**